src/server/status/metrics.js

```diff
@@ -1,27 +1,42 @@
 import _ from 'lodash';
 import Samples from './samples';
-module.exports = function (kbnServer, server, config) {
-  let lastReport = Date.now();
-
-  kbnServer.metrics = new Samples(12);
+import { keysToSnakeCaseShallow } from '../../utils/case_conversion';
 
+export function collectMetrics(kbnServer, server, config) {
   server.plugins['even-better'].monitor.on('ops', function (event) {
-    const now = Date.now();
-    const secSinceLast = (now - lastReport) / 1000;
-    lastReport = now;
-
-    const port = config.get('server.port');
-    const requests = _.get(event, ['requests', port, 'total'], 0);
-    const requestsPerSecond = requests / secSinceLast;
-
-    kbnServer.metrics.add({
-      heapTotal: _.get(event, 'psmem.heapTotal'),
-      heapUsed: _.get(event, 'psmem.heapUsed'),
-      load: event.osload,
-      responseTimeAvg: _.get(event, ['responseTimes', port, 'avg']),
-      responseTimeMax: _.get(event, ['responseTimes', port, 'max']),
-      requestsPerSecond: requestsPerSecond
-    });
-
+    kbnServer.metrics = getMetrics({ event, config });
   });
-};
+}
+
+export function getMetrics({ event, config }) {
+  const port = config.get('server.port');
+  const timestamp = new Date().toISOString();
+  return {
+    last_updated: timestamp,
+    collection_interval_in_millis: config.get('ops.interval'),
+    uptime_in_millis: process.uptime() * 1000,
+    process: {
+      mem: {
+        heap: {
+          total_in_bytes: _.get(event, 'psmem.heapTotal'),
+          used_in_bytes:  _.get(event, 'psmem.heapUsed')
```

Comment:
Can we change this to `heap_used_in_bytes` and nest under `mem` to align with ES?